packages/cloud_security_posture/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.0-next"
```

```suggestion
- version: "1.3.0-next"
- version: "1.3.0-preview"
```

We've found out that using `next` is not optimal since it is not updatable
Can you switch to `switch`